test/e2e/host_path.go

```diff
@@ -57,7 +57,7 @@ var _ = Describe("hostDir", func() {
 		}
 	})
 
-	It("MOD volume on tmpfs should have the correct mode", func() {
+	It("should give a MOD volume on tmpfs the correct mode", func() {
```

Comment:
This isn't an issue with your change, but I think this description is wrong.  The test really has nothing to do with tmpfs.